pkg/resources/management.cattle.io/v3/globalrole/validator.go

```diff
@@ -93,11 +99,6 @@ func (a *admitter) Admit(request *admission.Request) (*admissionv1.AdmissionResp
 			// This is needed to supported the deletion of old GlobalRoles that would not pass the update check that verifies all rules have verbs.
 			return admission.ResponseAllowed(), nil
 		}
-		if isMetaOnlyChange(oldGR, newGR) {
```

Comment:
This was implemented so that Rancher's controllers could add labels/annotations related to the controllers without bypassing the webhook.

Since rancher has * on * in * groups, the bind/escalate verb allows us to work without checking for metadata only changes, which is expensive and hacky. As such, this was removed.